From 32c9d01fe0168366b7c475ff3c9a574692917856 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Mon, 27 Aug 2012 15:17:21 +1000
Subject: [PATCH 2/3] config/udev: respect seat for hotplugged video devices.

This respects the seat tag for hotplugged video devices at X start.

Signed-off-by: Dave Airlie <airlied@redhat.com>
---
 config/udev.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/config/udev.c b/config/udev.c
index adac273..89b6e57 100644
--- a/config/udev.c
+++ b/config/udev.c
@@ -454,6 +454,10 @@ config_udev_odev_probe(config_odev_probe_proc_ptr probe_callback)
 
     udev_enumerate_add_match_subsystem(enumerate, "drm");
     udev_enumerate_add_match_sysname(enumerate, "card[0-9]*");
+#ifdef HAVE_UDEV_ENUMERATE_ADD_MATCH_TAG
+    if (ServerIsNotSeat0)
+        udev_enumerate_add_match_tag(enumerate, SeatId);
+#endif
     udev_enumerate_scan_devices(enumerate);
     devices = udev_enumerate_get_list_entry(enumerate);
     udev_list_entry_foreach(device, devices) {
-- 
1.7.10.2

